src/bin/common/parse_command.ml

```diff
@@ -1238,8 +1303,16 @@ let parse_fmt_opt =
     Arg.(value & opt formatter_conv Stderr & info ["err-formatter"] ~docs ~doc)
   in
 
+  let model_output =
+    let doc = Format.sprintf
+        "Set the model formatter used by default to output model and
```

Comment:
A better doc string would be something like: "Set the output used for the model generation. Possible values are ..."